runtime/doc/eval.txt

```diff
@@ -4244,7 +4212,7 @@ Textlock							*textlock*
 In a few situations it is not allowed to change the text in the buffer, jump
 to another window and some other things that might confuse or break what Vim
 is currently doing.  This mostly applies to things that happen when Vim is
-actually doing something else.  For example, evaluating the 'balloonexpr' may
+actually doing something else.  For example, evaluating the 'formatexpr' may
 happen any moment the mouse cursor is resting at some position.
```

Comment:
oops, this is nonsensical because the mouse hover was specifically referring to how balloonexpr works. 'formatexpr' is not eval'd on mouse hover. Here's a valid example:
```suggestion
For example, a TextYankPost autocmd cannot edit the text it is yanking. 
```